pygitguardian/client.py

```diff
@@ -514,3 +516,29 @@ def read_metadata(self) -> Optional[Detail]:
 
         self.secret_scan_preferences = metadata.secret_scan_preferences
         return None
+
+    def create_jwt(
+        self,
+        jwt_audience: str,
+        service: JWTService,
+        extra_headers: Optional[Dict[str, str]] = None,
+    ) -> Union[Detail, JWTResponse]:
+        """
+        Create a JWT token for other GitGuardian services.
+
+        Only used to interact with HasMySecretLeaked for now.
```

Comment:
There is a risk this comment won't be removed when we add more services :). I think it's safer to add it to the changelog fragment.